diff -urNp -x '*.orig' monitoring-plugins-2.3.org/plugins-root/Makefile.am monitoring-plugins-2.3/plugins-root/Makefile.am
--- monitoring-plugins-2.3.org/plugins-root/Makefile.am	2020-12-09 22:38:01.000000000 +0100
+++ monitoring-plugins-2.3/plugins-root/Makefile.am	2021-10-05 22:41:01.104964271 +0200
@@ -46,10 +46,6 @@ INSTALL_SUID = \
 	p=$$f; \
 	echo " $(INSTALL_PROGRAM) $$p $(DESTDIR)$(libexecdir)/$$p"; \
 	$(INSTALL_PROGRAM) $$p $(DESTDIR)$(libexecdir)/$$p; \
-	echo " chown root $(DESTDIR)$(libexecdir)/$$p"; \
-	chown root $(DESTDIR)$(libexecdir)/$$p; \
-	echo " chmod $(setuid_root_mode) $(DESTDIR)$(libexecdir)/$$p"; \
-	chmod $(setuid_root_mode) $(DESTDIR)$(libexecdir)/$$p; \
 	done
 
 install-root: $(noinst_PROGRAMS)
